src/components/missions.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable consistent-return */
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Mission from './mission';
+import { loadMisionsThunk } from '../redux/missions/missionReducer';
+import './mission.css';
+
+function Missions() {
+  const state = useSelector((state) => state.missionReducer);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (state.length === 0) {
+      dispatch(loadMisionsThunk());
+    }
+  }, []);
+  console.log(state);
```

Comment:
- [ ] Let's remove any `console.log` statement that we might have in the code. These are good for debugging, but should not be committed to production code.

Reply:
Thank you for this.
All console.log statements have been removed.

Reply:
Done ✔️